t/062-count.t

```diff
@@ -283,7 +283,7 @@ n = 4
 --- request
 GET /test
 --- response_body
-n = 18
+n = 19
```

Comment:
We should change the initial Lua table size accordingly otherwise these tests make no sense.